web/ui/react-app/src/ExpressionInput.tsx

```diff
@@ -95,63 +96,68 @@ class ExpressionInput extends Component<ExpressionInputProps> {
 
   render() {
     return (
-        <Downshift
-          //inputValue={this.props.value}
-          //onInputValueChange={this.props.onChange}
-          selectedItem={this.props.value}
-        >
-          {(downshift) => (
-            <div>
-              <InputGroup className="expression-input">
-                <InputGroupAddon addonType="prepend">
-                  <InputGroupText>
-                  {this.props.loading ? <FontAwesomeIcon icon="spinner" spin/> : <FontAwesomeIcon icon="search"/>}
-                  </InputGroupText>
-                </InputGroupAddon>
-
-                <Input
-                  autoFocus
-                  type="textarea"
-                  rows="1"
-                  onKeyPress={this.handleKeyPress}
-                  placeholder="Expression (press Shift+Enter for newlines)"
-                  innerRef={this.exprInputRef}
-                  {...downshift.getInputProps({
-                    onKeyDown: (event: React.KeyboardEvent): void => {
-                      switch (event.key) {
-                        case 'Home':
-                        case 'End':
-                          // We want to be able to jump to the beginning/end of the input field.
-                          // By default, Downshift otherwise jumps to the first/last suggestion item instead.
+      <Downshift
```

Comment:
Did anything change in this block besides formatting? It's hard to see with the whole block being reindented.